modules/registry-exposing-default-registry-manually.adoc

```diff
@@ -0,0 +1,51 @@
+[id="registry-exposing-default-registry-manually_{context}"]
+= Exposing a default registry manually
+
+Instead of logging in to the default {product-title} registry from within the cluster, you can gain external access to it by exposing it with a route. This external access allows you to log in to the registry from outside the cluster using the route address and to tag and push images to an existing project by using the route host.
```

Comment:
```suggestion
Instead of logging in to the default {product-title} registry from within the cluster, you can gain external access to it by exposing it with a route. This external access enables you to log in to the registry from outside the cluster using the route address and to tag and push images to an existing project by using the route host.
```

IBM Style Guide:

> Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."